pypxe-server.py

```diff
@@ -29,6 +33,14 @@
 DHCP_BROADCAST = '<broadcast>'
 DHCP_FILESERVER = '192.168.2.2'
 
+#Service bools
```

Comment:
I'm not sure I understand why these lines were added, I thought the defaults were handled in `add_argument`?